tests/test_s_bend.py

```diff
@@ -144,7 +144,7 @@ def df_trajectory(x, y, z):
 
    trajectory = rqw.Trajectory(model, md_datum = datum, data_frame = df, length_uom = 'm', well_name = 'ANGLED_WELL')
 
-   rqc.Crs(model, trajectory.crs_root).uuid == crs.uuid
+   rqc.Crs(model, uuid=trajectory.uuid).uuid == crs.uuid
```

Comment:
This doesn't look right. The trajectory has its own uuid (trajectory.uuid) and it has a reference to a crs object. For historical reasons that is currently trajectory.crs_root (but needs at some point to be changed to trajectory.crs_uuid).  For now, I think we need something like:
   rqc.Crs(model, uuid = rqet.uuid_for_part_root(trajectory.crs_root)).uuid == crs.uuid